src/test/e2e/bundle_test.go

```diff
@@ -30,7 +31,7 @@ func zarfPublish(t *testing.T, path string, reg string) {
 	require.NoError(t, err)
 }
 
-const zarfVersion = "v0.29.1"
+const zarfVersion = "v0.30.1"
```

Comment:
this should prob be an env var that gets passed into `go test`

Reply:
Agreed, made an issue to track: https://github.com/defenseunicorns/uds-cli/issues/126